builtin/providers/aws/resource_aws_sqs_queue.go

```diff
@@ -177,6 +178,9 @@ func resourceAwsSqsQueueRead(d *schema.ResourceData, meta interface{}) error {
 					}
 					d.Set(iKey, value)
 				} else {
+					if iKey == "policy" {
+						*attrmap[oKey] = normalizeJson(*attrmap[oKey])
+					}
```

Comment:
I found opened pull-request, #4780.

And, I stopped use here document SQS policy.